index.js

```diff
@@ -1,32 +1,62 @@
-function ratioToRGB ({ r, g, b }) {
+function ratioToRGB({ r, g, b }) {
   return ({
     r: r * 255,
     g: g * 255,
-    b: b * 255
+    b: b * 255,
   })
 }
 
-export default function getColors (document, nodeType, nameRegex) {
+function ratioToRGBA({ r, g, b, a }) {
+  return ({
+    r: r * 255,
+    g: g * 255,
+    b: b * 255,
+    a: a
+  })
+}
+
+export default function getColors(document, channelType, nodeType, nameRegex) {
```

Comment:
Can you please move the `channelType` parameter to the end? This way we won't break any existing code